### DISCUSSION TITLE
Date Picker filled in some browsers and empty in some

### DISCUSSION THREAD

**Discussion by ehsaan-changa** [twilio-labs/paste#3290] [Q&A] [answered]
How to make date picker empty across all browsers. By default date picker is already filled in some browsers (eg :safari)  and empty in some browsers (eg : chrome).

My use case is i want to enable my submit button when all the fields are filled , but when it comes to date picker its already filled in some browsers and empty in some .

**Comment by SiTaggart** [comment, marked as answer]
Hi there @ehsaan-changa 

If you are validating the value of the date picker component itself, regardless of cross browser appearance, you should find the value is actually empty.

For best practices, though, we don't actually recommend what you are trying to do, which is disabling the button prior to submission.  It's much better to communicate the requirements more explicitly using help text, error messaging and validation. 

More context can be found in our [Form validation guidelines](https://paste.twilio.design/components/form#validation) and [disabled button docs](https://paste.twilio.design/components/button#disabled).

**Reply by ehsaan-changa** [reply]
Hey @SiTaggart 

Thanks for the feedback 

Can you clear me below question as well 

How to make date picker empty across all browsers. By default date picker is already filled in some browsers (eg :safari) and empty in some browsers (eg : chrome). 

Is it possible or not ??

**Reply by SiTaggart** [reply]
We can't control the way the native browser input works, unfortunately. For whatever reason, Safari decided to ship with what is effectively a placeholder for today's date. It's not actually the value though, it's still technically empty.

**Reply by ehsaan-changa** [reply]
Ok Sure Thanks
